File: sample/src/main/java/com/example/myapplication/MainActivity.java
package com.example.myapplication;

import android.content.Context;
import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;
import androidx.constraintlayout.widget.ConstraintLayout;

import com.softeq.bottomsheet.AnchorBottomSheetBehavior;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ConstraintLayout constraintLayout = this.findViewById(R.id.bottomSheet);
        AnchorBottomSheetBehavior behavior = AnchorBottomSheetBehavior.from(constraintLayout);

        int hPx = getResources().getDisplayMetrics().heightPixels;
        float cardAnchorSizeDp = 270 + 20;
        float cardAnchorSizePx = pxFromDp(this, cardAnchorSizeDp);

        int anchor = hPx - (int) cardAnchorSizePx;

        behavior.setAnchorOffset(anchor);
    }

    public static float dpFromPx(final Context context, final float px) {
        return px / context.getResources().getDisplayMetrics().density;
    }

    public static float pxFromDp(final Context context, final float dp) {
        return dp * context.getResources().getDisplayMetrics().density;
    }
}

File: samples/MainActivity.cs
using Android.App;
using Android.Content;
using Android.OS;
using AndroidX.AppCompat.App;
using AndroidX.ConstraintLayout.Widget;
using Com.Trafi.Anchorbottomsheetbehavior;

namespace AnchorBottomSheetBehaviorSamples;

[Activity(
    Label = "@string/app_name",
    Theme = "@style/AppTheme.NoActionBar",
    MainLauncher = true)]
public class MainActivity : AppCompatActivity
{
    protected override void OnCreate(Bundle savedInstanceState)
    {
        base.OnCreate(savedInstanceState);
        SetContentView(Resource.Layout.activity_main);

        ConstraintLayout constraintLayout = FindViewById<ConstraintLayout>(Resource.Id.bottomSheet);
        AnchorBottomSheetBehavior behavior = AnchorBottomSheetBehavior.From(constraintLayout);

        int hPx = Resources.DisplayMetrics.HeightPixels;
        float cardAnchorSizeDp = 270 + 20;
        float cardAnchorSizePx = PxFromDp(this, cardAnchorSizeDp);

        int anchor = hPx - (int)cardAnchorSizePx;

        behavior.AnchorOffset = anchor;
    }

    public static float DpFromPx(Context context, float px)
    {
        return px / context.Resources.DisplayMetrics.Density;
    }

    public static float PxFromDp(Context context, float dp)
    {
        return dp * context.Resources.DisplayMetrics.Density;
    }
}
